.github/workflows/check-dist.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Setup Node.js 20.x
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
```

Comment:
I'm not sure if this would require a major version bump, as touched upon in https://github.com/Swatinem/rust-cache/issues/183.